mullvad-cli/src/cmds/patch.rs

```diff
@@ -99,3 +117,30 @@ fn read_settings_from_reader(mut reader: impl BufRead) -> Result<String> {
         .context("settings must be utf8 encoded")?
         .to_owned())
 }
+
+fn put_blob(dest: String, blob: String) -> Result<()> {
+    match dest.as_str() {
+        "-" => write_settings_to_stdout(blob).context("Failed to write to stdout"),
+        _ => write_settings_to_file(&dest, blob).context(format!("Failed to write to path {dest}")),
+    }
+}
+
+/// Write patch to standard output
+fn write_settings_to_stdout(blob: String) -> Result<()> {
+    write_settings_using_writer(BufWriter::new(stdout()), blob)
+}
+
+/// Write patch to path
+fn write_settings_to_file(path: impl AsRef<Path>, blob: String) -> Result<()> {
+    write_settings_using_writer(
+        BufWriter::new(File::options().create(true).write(true).open(path)?),
+        blob,
+    )
+}
+
+fn write_settings_using_writer(mut writer: impl Write, blob: String) -> Result<()> {
+    writer
+        .write_all(blob.as_bytes())
+        .context("Failed to write blob to destination")?;
+    writer.write_all(b"\n").context("Failed to write newline")
+}
```

Comment:
Why do we need to use a `BufWriter` here? Look at the documentation it says "BufWriter<W> can improve the speed of programs that make small and repeated write calls to the same file or network socket. It does not help when writing very large amounts at once, or writing just one or a few times. It also provides no advantage when writing to a destination that is in memory, like a Vec<u8>".

Writing the settings a single time doesn't seem like the right use case to me.

Reply:
Nope, I don't disagree. :) I guess one `write` syscall is slightly better than two (due to the newline)? But it really does not matter, so I'll remove it.

Reply:
Done.